lte/gateway/python/magma/subscriberdb/protocols/m5g_auth_servicer.py

```diff
@@ -14,12 +14,17 @@
 import logging
 
 from google.protobuf.json_format import MessageToJson
+from lte.protos.subscriberauth_pb2 import M5GAuthenticationInformationAnswer
```

Comment:
It passes the lte test if we use from (lte.protos import subscriberauth_pb2_grpc, subscriberauth_pb2).